invenio_rdm_pure/ext.py

```diff
@@ -41,14 +41,18 @@ def init_config(self, app):
             ):
                 app.config.setdefault(k, getattr(config, k))
         if "SQLALCHEMY_DATABASE_URI" in app.config:
-            database_uri = app.config.get("SQLALCHEMY_DATABASE_URI").split("//")[1]
-            database_credentials = database_uri.split("@")[0]
-            database_connection = database_uri.split("@")[1]
-            database_host = database_connection.split("/")[0]
-            database_name = database_connection.split("/")[1]
-            database_username = database_credentials.split(":")[0]
-            database_password = database_credentials.split(":")[1]
-            app.config.setdefault("INVENIO_DATABASE_HOST", database_host)
-            app.config.setdefault("INVENIO_DATABASE_NAME", database_name)
-            app.config.setdefault("INVENIO_DATABASE_USERNAME", database_username)
-            app.config.setdefault("INVENIO_DATABASE_PASSWORD", database_password)
+            if (
+                len(app.config.get("SQLALCHEMY_DATABASE_URI").split("//")) > 1
+                and len(app.config.get("SQLALCHEMY_DATABASE_URI").split("//")[1]) > 0
+            ):
+                database_uri = app.config.get("SQLALCHEMY_DATABASE_URI").split("//")[1]
+                database_credentials = database_uri.split("@")[0]
+                database_connection = database_uri.split("@")[1]
+                database_host = database_connection.split("/")[0]
+                database_name = database_connection.split("/")[1]
+                database_username = database_credentials.split(":")[0]
+                database_password = database_credentials.split(":")[1]
+                app.config.setdefault("INVENIO_DATABASE_HOST", database_host)
+                app.config.setdefault("INVENIO_DATABASE_NAME", database_name)
+                app.config.setdefault("INVENIO_DATABASE_USERNAME", database_username)
+                app.config.setdefault("INVENIO_DATABASE_PASSWORD", database_password)
```

Comment:
Question: do we use these variables anywhere in the module?
as I remember we needed in these [lines of code](https://github.com/tu-graz-library/invenio-rdm-pure/blob/027290b4bfcb5d8466daf7190767030fc7cc1309/invenio_rdm_pure/source/rdm/database.py#L30), but since it's commented out and if we do not need them anymore- you can get rid of them.

Reply:
Also good point.. 7cd0156110b0fd9bd1f0f82b659526a235c5f21c removes the initialization, and 1cec02fdb81c4394844a21f70092c6a0d8934787 removes their *unused* references.